### DISCUSSION TITLE
Use permanent volume for Reports  & Data ?  (e.g. --volume gvm-data:/data)

### DISCUSSION THREAD

**Discussion by 4920441** [Secure-Compliance-Solutions-LLC/GVM-Docker#185] [Q&A] [answered]
Hi,
when I run the container with podman I simply add 

` --volume gvm-data:/data`

I get an Error that the postresql.conf is not present.

Is there currently a way to use a permanent data volume for all reports and config stuff?

In the wiki is nothing mentioned any more ( but I was sure, there was in the recent past...)

Starting it with 

`podman run -it -p 8080:9392 -e PASSWORD="securepassword" docker.io/securecompliance/gvm:21.4.0-v2
`
certainly results in a clean restart without any reports and definitions.

Thanks a lot!

Cheers

4920441

**Comment by karlbrown87** [comment, marked as answer]
Docker volume create gvm-data

Obtener Outlook para Android<https://aka.ms/ghei36>

________________________________
From: 4920441 ***@***.***>
Sent: Thursday, May 27, 2021 9:42:28 AM
To: Secure-Compliance-Solutions-LLC/GVM-Docker ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [Secure-Compliance-Solutions-LLC/GVM-Docker] Use permanent volume for Reports & Data ? (e.g. --volume gvm-data:/data) (#185)


Hi,
when I run the container with podman I simply add

--volume gvm-data:/data

I get an Error that the postresql.conf is not present.

Is there currently a way to use a permanent data volume for all reports and config stuff?

In the wiki is nothing mentioned any more ( but I was sure, there was in the recent past...)

Starting it with

podman run -it -p 8080:9392 -e PASSWORD="securepassword" docker.io/securecompliance/gvm:21.4.0-v2
certainly results in a clean restart without any reports and definitions.

Thanks a lot!

Cheers

4920441

**Reply by 4920441** [reply]
Yeah, thanks for the hint.... It was a PEBKAC*) problem 

I had an old data volume with the same name and a very old config already existent, so that was the cause for the many problems I got....

Sorry bothering you all:-)

Cheers

4920441




*)PEBKAC - Problem Exists Between Keyboard And Chair

**Comment by austinsonger** [comment]
> Hi,
> when I run the container with podman I simply add
> 
> ` --volume gvm-data:/data`
> 
> I get an Error that the postresql.conf is not present.
> 
> Is there currently a way to use a permanent data volume for all reports and config stuff?
> 
> In the wiki is nothing mentioned any more ( but I was sure, there was in the recent past...)
> 
> Starting it with
> 
> `podman run -it -p 8080:9392 -e PASSWORD="securepassword" docker.io/securecompliance/gvm:21.4.0-v2 `
> certainly results in a clean restart without any reports and definitions.
> 
> Thanks a lot!
> 
> Cheers
> 
> 4920441

Please don't use the wiki. The link to the documentation is in the description.

https://securecompliance.gitbook.io/projects/openvas-greenbone-deployment-full-guide

**Reply by 4920441** [reply]
Erm.... you saw what I wrote I did it totally the right way was only unable to see that I was recycling an old data volume so old and obsolete config fies were used?

(sarkasm_on )And i read never the F* manual and never read any wikis! I go to hell if  I do so.....   (sarkasm_off)
